style.css

```diff
@@ -3227,15 +3227,32 @@ a.header-image img {
 	.menu-primary > .menu-wrap > li > a {
 		font-weight: bold;
 		line-height: 2.5;
+<<<<<<< HEAD
```

Comment:
I don't know what these are called but there seems to be a bunch of merge conflict messages committed here :)

Running npm run build will rebuild the css and remove this.

Reply:
I'm not sure why there are merge conflicts. I updated the PR with your latest changes in Master.